stable/appmesh-controller/Chart.yaml

```diff
@@ -1,6 +1,19 @@
-apiVersion: v2
+apiVersion: v1
 name: appmesh-controller
-description: The AWS App Mesh controller Helm chart for Kubernetes
-type: application
+description: App Mesh controller Helm chart for Amazon EKS
 version: 0.2.0
 appVersion: 0.2.0
+home: https://github.com/aws/eks-charts
+icon: https://raw.githubusercontent.com/aws/eks-charts/master/docs/logo/amazon-eks.png
```

Comment:
After second thoughts, it would be best to have this be the AWS logo, not the EKS one, since these charts should be for all k8s on AWS, not just EKS. I'll find you an appropriate square transparent AWS logo, Stefan.